### PR TITLE
Feat: Like Page added

### PR DESCRIPTION
**Pull request by rahul31agrawal**
Hi Folks,
I have added like page to my Video library.
Please give your PR reviews

### CODE REVIEW

File: src/components/sideBar/sideBar.js [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 
 const Sidebar = () => {
-
+  const navigate = useNavigate();
   return(
     <aside className = "sideBar">
     <div className = "sidebarContents">
       <i className="fas fa-home fa-2x"></i>
-      <h3 className = "sideBarText">Home</h3>
+      <h3 className = "sideBarText" onClick={() => { navigate("/")}}>Home</h3>
```
Comment (sudosohan) [inline src/components/sideBar/sideBar.js:9]: You can use `<NavLink />`  here instead of `useNavigate`

**Review by sohan9819 — APPROVED** [review]
Everything looks good . Prefer using `<NavLink />` for navigation links in the side bar or in the nav . Using this you also show the active section to the user .

File: src/App.js [hunk 2]
```diff
@@ -2,13 +2,14 @@ import "./App.css";
...
 import {Home} from "./pages/homePage/home";
-
+import {LikesPage} from "./pages/likesPage/likesPage";
 function App() {
   return (
     <div className="App">
       <Routes>
         <Route path="/" element={<Home />} />
         <Route path="/Videos" element={<VideoListing/>} />
+        <Route path="/LikesPage" element={<LikesPage/>} />
```
Comment (vishalg8454) [inline src/App.js:12]: why using upper case in path?

**Review by vishalg8454 — COMMENTED** [review]
Good to merge. Can look at the comment I left, also remove the white spaces from the code.

**Pull request merged**